src/Powercord/coremods/moduleManager/commands/reload.js

```diff
@@ -0,0 +1,60 @@
+const { resp } = require('../util');
+
+module.exports = {
+  command: 'reload',
+  description: 'Reload a plugin/theme',
+  usage: '{c} [ plugin/theme ID ]',
+  executor ([ id ]) {
+    const isPlugin = powercord.pluginManager.plugins.has(id);
+    const isTheme = powercord.styleManager.themes.has(id);
+
+    if (!isPlugin && !isTheme) { // No match
+      return resp(false, `Could not find plugin or theme matching "${id}".`);
+    } else if (isPlugin && isTheme) { // Duplicate name
+      return resp(false, `"${id}" is in use by both a plugin and theme. You will have to reload it from settings.`);
+    } else if (isPlugin) { // Core internal plugin
+      return resp(false, `"${id}" provides core functionality and cannot be reloaded.`);
+    }
+
+    const manager = isPlugin ? powercord.pluginManager : powercord.styleManager;
+    if (!manager.isEnabled(id)) {
+      return resp(false, `"${id}" is already disabled.`);
+    }
+
+    manager.disable(id);
```

Comment:
Using `manager.remount(id)` will automatically remount it for plugins. This isn't the same as enabling and disabling as it 'takes it off' of discord and then back on.
You'll have to make a similar one for themes.